pkg/cache/postgres.go

```diff
@@ -54,7 +54,7 @@ func (c *postgresCache) AddCheckFromStatus(check pkg.Check, status pkg.CheckStat
 		return uuid.Nil, nil
 	}
 
-	if !check.Transformed {
+	if check.ID != uuid.Nil {
```

Comment:
@yashmehrotra How will this affect the transformed canaries ?

Reply:
It will work fine, since transformed checks won't have the check id set

@adityathebe Can you confirm by running the alertmanager check

Reply:
@moshloop @yashmehrotra can confirm that alertmanager checks were created.